composer.json

```diff
@@ -28,6 +28,7 @@
             "dev-master": "1.x-dev"
         },
         "expose": [
+            "client/img",
```

Comment:
You could also put the img folder under client/dist

Reply:
I could, but they're not generated. The images are both the source and the dist.

Reply:
dist = distributable, they don't need to be generated. If you feel strongly then it's fine to leave it where it is though